lib/ErrorSerializer.js

```diff
@@ -0,0 +1,20 @@
+var serializeError = require('serialize-error');
```

Comment:
Actually I'll take that back. It seems JSONSerializer doesn't account for circular references, which I assumed it does.